### PR TITLE
feat / xemm documentation

### PR DESCRIPTION
**Pull request by martinkou**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:
https://app.clubhouse.io/coinalpha/story/3806/document-a-high-level-description-of-the-internal-logic-of-strategies

**A description of the changes proposed in the pull request**:
This patch adds development documentation for the cross exchange market making strategy.

### CODE REVIEW

**Comment by martinkou** [comment]
ok to test

File: documentation/docs/developers/strategies/cross-exchange-market-making.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+When `active_order_canceling` is enabled, the cross exchange market making strategy would refresh orders by actively cancelling them regularly. This is optimal for centralized exchanges because it allows the strategy to respond quickly when, for example, market prices have significantly changed. This should not be chosen for decentralized exchanges, because actively cancelling orders on DEXes can often take a long time and may also require gas cost.
+
+When `active_order_canceling` is disbled, the cross exchange market making strategy would emit limit orders that automatically expire after a predefined time period. This means the strategy can just wait for them to expire to refresh the maker orders, rather than having to cancel them actively. This is useful for decentralized exchanges because it avoids the potentially very long cancellation delays there, and it also does not cost any gas to wait for order expiration.
```
Comment (mifeng) [inline documentation/docs/developers/strategies/cross-exchange-market-making.md]: disbled => disabled

File: documentation/docs/developers/strategies/cross-exchange-market-making.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+![Figure 2: Cancel order flow chart](/assets/img/xemm-flowchart-2.svg)
+
+#### Active order cancellation setting
+
+The [`active_order_canceling`](/strategies/corss-exchange-market-making/) setting changes how the cancel order flow operates. `active_order_canceling` should be enabled when the maker side is a centralized exchange (e.g. Binance, Coinbase Pro), and it should be disabled when the maker side is a decentralized exchange.
+
+When `active_order_canceling` is enabled, the cross exchange market making strategy would refresh orders by actively cancelling them regularly. This is optimal for centralized exchanges because it allows the strategy to respond quickly when, for example, market prices have significantly changed. This should not be chosen for decentralized exchanges, because actively cancelling orders on DEXes can often take a long time and may also require gas cost.
```
Comment (mifeng) [inline documentation/docs/developers/strategies/cross-exchange-market-making.md]: "This should not be chosen for decentralized exchanges that charge gas for cancelling orders (such as Radar Relay
)"

**Comment by martinkou** [comment]
@veecos2 I'm avoiding discussing the details to how the price and sizes are determined because I expect you're making those changes soon. I'd say we should push the current docs out first, and then fill in the details of the create order section once you've got your new XEMM logic (https://app.clubhouse.io/coinalpha/story/5132/change-xemm-to-always-place-orders-deep-in-the-order-book) completed.

**Pull request merged**